add deepseek-v3 amd docker command (#3495)
sgl-project/sglang: docs/references/amd.md

```

With your AMD system properly configured and SGLang installed, you can now fully leverage AMD hardware to power SGLang’s machine learning capabilities.

## Running DeepSeek-V3

> To run DeepSeek-V3 the only difference is when launching the server.

Reviewer: ```suggestion
The only difference in running DeepSeek-V3 is when starting the server.
```
